internal/v1/handler_compose_image.go

```diff
@@ -830,6 +830,29 @@ func buildCustomizations(cust *Customizations) (*composer.Customizations, error)
 		}
 	}
 
+	// we need to explicitly add 'rhcd' to the enabled services
+	// if openscap and subscription customizations are set, otherwise
+	// the insights-client doesn't register properly
+	if cust.Subscription != nil && cust.Subscription.Insights && cust.Openscap != nil {
+		if res.Services == nil {
+			res.Services = &composer.Services{}
+		}
+		if res.Services.Enabled == nil {
+			res.Services.Enabled = &[]string{}
+		}
+		containsService := func(services []string, service string) bool {
+			for _, s := range services {
+				if s == service {
+					return true
+				}
+			}
+			return false
+		}
```

Comment:
Might make sense to add comment `TODO: use slices.Contains(services, "rhcd") when migrating to go 1.21`

Reply:
I half considered it. I think we should add it now. I'll quickly do that and bug you for a review